beetsplug/subsonicupdate.py

```diff
@@ -29,23 +29,32 @@
 
 import requests
 
+from binascii import hexlify
 from beets import config
 from beets.plugins import BeetsPlugin
 
 __author__ = 'https://github.com/maffo999'
+authtokenversion = '1.12'
```

Comment:
FWIW, it's generally nice for module-level constants to get `CAPITAL_NAMES`, like `AUTH_TOKEN_VERSION`.